src/containers/Settings/index.js

```diff
@@ -1,184 +1,116 @@
 import React, { useState } from "react";
-import { useTable } from 'react-table'
 import AppPage from "../AppPage/index";
 import { connect } from "react-redux";
-import { updateBudget } from "../../redux/actions/auth";
-import { useHistory } from "react-router-dom";
-import styled from 'styled-components';
-
-// Color Palette
-// Dark Pink: #FF3576
-// Medium pink: #FF8FB3
-// Lighter pink: #F9DAE4
-// Sidebar pink: #FDF6F8
-// Grey: #EEEEEE
-
-// Circle: #FFAEC8
-// Lighter circle: #F9DAE4
-
-// Dark line: #AB4A78
-
-const SettingPage = ({ user, updateBudget }) => {
-   const history = useHistory()
-   const [value, setValue] = useState({
-      client_id: user ? user.id : null,
-      currentBudget: "",
-      budgets: [
-         { month: 8, budget: 5000.00, spendings: 5510.00},
-         { month: 9, budget: 4000.00, spendings: 4567.89},
-         { month: 10, budget: 2500.00, spendings: 2500.00},
-         { month: 11, budget: 1500.00, spendings: 1000.00}
-      ],
+import {
+  Grid,
+  Container,
+  FormControl,
+  Input,
+  InputLabel,
+  InputAdornment,
+  makeStyles,
+
+  Button,
+} from "@material-ui/core";
+import BudgetHistory from "./BudgetHistory";
+import { createBudget} from "../../redux/actions/budget";
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    flexWrap: "wrap",
+  },
+  margin: {
+    margin: theme.spacing(1),
+  },
+  withoutLabel: {
+    marginTop: theme.spacing(3),
+  },
+  textField: {
+    width: "25ch",
+  },
+}));
+
+const SettingPage = ({ user, createBudget }) => {
+  const classes = useStyles();
+  const [values, setValues] = React.useState({
+    amount: "",
+  });
+
+
+  const handleSubmit = () => {
```

Comment:
Good work fetching data from the backend! Good use of functions to handle events